Makefile

```diff
@@ -27,60 +27,20 @@ certificates/domain.key:
 
 .PHONY: up-local
 up-local: .install-fqdn certificates/domain.crt certificates/domain.key .create-secrets ## deploy osparc ops stacks and simcore, use minio_disabled=1 if minio s3 should not be started (if you have custom S3 set up)
-	@bash scripts/deployments/deploy.sh --stack_target=local --minio_enabled=0 --vcs_check=1
+	@bash scripts/deployments/deploy_everything.bash --stack_target=local --minio_enabled=0 --vcs_check=1
```

Comment:
`deploy_everything_locally.bash`?

Reply:
true that is better I will rename

Reply:
ah no even worse you spotted a bug, thanks man